packages/start-slicemachine/src/lib/createSliceMachineServer.ts

```diff
@@ -79,6 +85,22 @@ export const createSliceMachineServer = async (
 
 	router.add("/api/t", eventHandler(sentryHandler));
 
+	router.add(
+		"/api/crash",
+		eventHandler(() => {
+			throw new Error("Uncaught Error");
+		}),
+	);
+	router.add(
+		"/api/error",
+		eventHandler(() => {
+			return createError({
+				status: 500,
+				message: "Test Error",
+			});
+		}),
+	);
+
```

Comment:
Is this endpoint still needed? `"Test Error"` sounds like something used during development.

Reply:
Same as above @mathieud?